src/main/java/com/example/auctionapp/repository/ProductRepository.java

```diff
@@ -174,7 +175,7 @@ public Double getCollectionLowestPrice(String categoryName) {
         Root<Product> resource = q.from(Product.class);
         q.select(resource);
 
-        Predicate predicateForEndDate = cb.greaterThanOrEqualTo(resource.<LocalDateTime>get("auctionEndDate"), LocalDateTime.now());
+        Predicate predicateForEndDate = cb.greaterThanOrEqualTo(resource.<LocalDateTime>get("auctionEndDate"), LocalDateTime.now(ZoneOffset.UTC));
```

Comment:
Just looking at this file we can see a potential method.
Create a static method in this file that returns this predicate.